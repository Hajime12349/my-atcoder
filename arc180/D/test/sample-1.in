7 5
4 3 1 1 4 5 2
1 7
2 4
3 5
1 5
4 7
