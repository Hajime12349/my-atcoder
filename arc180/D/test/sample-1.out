10
5
6
9
8
